templates/matchingDropDown.jsx

```diff
@@ -176,16 +176,19 @@ export default function MatchingDropDown(props) {
           !_isEnabled && 'is-disabled',
           hasActiveOption && 'is-selected'
         ])}
+        role="combobox"
         aria-haspopup="listbox"
         aria-expanded={isOpen}
+        aria-controls={`${_id}-matching-item-${_itemIndex}__list`}
         onMouseDown={onStartInteraction}
         onTouchStart={onStartInteraction}
         onClick={onButtonClick}
         ref={button}
-        aria-labelledby={props.questionTitleId + ' ' + props.questionTextId}
+        aria-labelledby={props.questionTitleId}
+        aria-activedescendant={highlightedOption && `dropdown__item__${_id}__${_itemIndex}__${highlightedOption._index}`}
```

Comment:
Moving aria-activedescendant from the listbox to the combobox is causing some strange behaviour for keyboard navigation/screen readers. Currently if I open the matching item the list items either not being read out (firefox) or missed entirely (chrome), both tested with NVDA. Is moving aria-activedescendant necessary?

Reply:
> Is moving aria-activedescendant necessary?
>
I don't think so. If I revert `aria-activedescendant` back to listbox, this reads as expected for NVDA on Chrome, Edge and FF. No change to VoiceOver on Safari and Chrome either. I'll commit this back.

Reply:
This is ready to retest in [98b84d6](https://github.com/adaptlearning/adapt-contrib-matching/pull/198/commits/98b84d6a13c7cd7480479a41f45cc8e28cd0dad0) please @joe-allen-89.